### PR TITLE
feat(pfd): add EIS standard S14 features

### PR DESCRIPTION
**Pull request by lukecologne**
Fixes #[issue_no]

## Summary of Changes

This PR adds EIS standard S14 features to the PFD. These include:
* Autobrake indications on the PFD
   * Active autobrake modes in the 1st line of the 1st column. These show the active A/BRK mode, if no A/THR mode is shown.
   * Armed autobrake modes, except BRK MAX in the 2nd line of the 1st column. These show the armed A/BRK mode, if no 2-line A/THR mode is active (i.e. MAN TOGA, MAN FLX etc.)
   * BRK MAX in the 3rd line of the 1st column. This is shown if the A/BRK MAX mode is armed, and if no other message (for example SPD SEL, MACH SEL, THR CLB etc.) is shown in this area.
   * DECEL indication below the speed tape. This has the same conditions as the DECEL light on the A/BRK buttons, and hides the lower selected speed text if it would be shown there.
* White selected altitude indicator, if a vertical AP/FD mode is active that ignores the selected altitude (i.e. G/S, G/S*, ROLL OUT, FLARE, LAND, FINAL APP)

The final change of S14 are the numbers next to the linear deviation indicator. These only make sense with VNAV, and are implemented already on experimental with the reworked PFD, so I've chosen not to add them in this PR.
Note that the BRK MAX indication on the third line is actually an addition of S15, on S14 this would be also on the second line.
I've chosen to already implement it like in S15.

Finally, the condition for classifying A/BRK as "active" is currently just DECEL being active. This is not correct, it should be a seperate signal, for example it is possible to have A/BRK not reach the target deceleration on a slippery runway, in which case the A/BRK mode should show in green, while DECEL is not shown. This is the reason this PR is still in draft, I'm awaiting this signal to be implemented.

## Screenshots (if necessary)

grafik
grafik
grafik

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/PFD/FMA.tsx [hunk 1]
```diff
@@ -12,6 +12,11 @@ export const FMA = ({ isAttExcessive }) => {
     const AB3Message = (getSimVar('L:A32NX_MachPreselVal', 'mach') !== -1
         || getSimVar('L:A32NX_SpeedPreselVal', 'knots') !== -1) && !BC3Message && engineMessage === 0;
 
+    // This is incorrect, A/BRK active should be a seperate signal and not just DECEL.
```
Comment (tracernz) [inline src/instruments/src/PFD/FMA.tsx:15]: Comment no longer relevant?

Comment (lukecologne) [reply src/instruments/src/PFD/FMA.tsx:15]: Oh yeah, forgot to remove that

**Review by tracernz — APPROVED** [review]
Great work.

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #6919
Tier of Testing  : 1
Date             : 16/03/2022

Testing Process: 
Autobrake indications on the PFD
Active autobrake modes in the 1st line of the 1st column. These show the active A/BRK mode, if no A/THR mode is shown. - **OK**
Armed autobrake modes, except BRK MAX in the 2nd line of the 1st column. These show the armed A/BRK mode, if no 2-line A/THR mode is active (i.e. MAN TOGA, MAN FLX etc.) - **OK**
BRK MAX in the 3rd line of the 1st column. This is shown if the A/BRK MAX mode is armed, and if no other message (for example SPD SEL, MACH SEL, THR CLB etc.) is shown in this area. - **OK**
DECEL indication below the speed tape. This has the same conditions as the DECEL light on the A/BRK buttons, and hides the lower selected speed text if it would be shown there. - **OK**
White selected altitude indicator, if a vertical AP/FD mode is active that ignores the selected altitude (i.e. G/S, G/S*, ROLL OUT, FLARE, LAND, FINAL APP) - **OK**

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image
image

**Comment by davidnorfolk96** [comment]
Quality Assurance Report

Discord: !davidn123#0001
Object of testing: 6919
Tier of Testing: 1
Date: 18/03/2022

Testing Process: Tested on a full flight & tested the following:

Active autobrake modes in the 1st line of the 1st column. These show the active A/BRK mode, if no A/THR mode is shown.-PASSED
Armed autobrake modes, except BRK MAX in the 2nd line of the 1st column. These show the armed A/BRK mode, if no 2-line A/THR mode is active (i.e. MAN TOGA, MAN FLX etc.)-PASSED
BRK MAX in the 3rd line of the 1st column. This is shown if the A/BRK MAX mode is armed, and if no other message (for example SPD SEL, MACH SEL, THR CLB etc.) is shown in this area.-PASSED
DECEL indication below the speed tape. This has the same conditions as the DECEL light on the A/BRK buttons, and hides the lower selected speed text if it would be shown there.-PASSED
White selected altitude indicator, if a vertical AP/FD mode is active that ignores the selected altitude (i.e. G/S, G/S*, ROLL OUT, FLARE, LAND, FINAL APP)-PASSED

Negatives: N/A

Testing Results: Passed

Conclusions: n/a

Media:
Screenshot_5
Screenshot_6

**Pull request merged**